src/main/java/com/DAO/TeamDAO.java

```
@@ -0,0 +1,72 @@
+package com.DAO;
+
+import com.entity.Team;
+import com.opencsv.CSVReader;
+import com.opencsv.CSVWriter;
+import com.opencsv.exceptions.CsvException;
+import org.springframework.stereotype.Repository;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+@Repository
+public class TeamDAO {
+private final String fileName="TeamsDB.csv";
+
+@PostConstruct
+private void init(){
+    File data=new File(fileName);
+    try {
+        data.createNewFile();
+    } catch (IOException e) {
+        e.printStackTrace();
+    }
+}
+public List<Team> getData(){
+    CSVReader reader=null;
+    List<String[]> strData = null;
+    List<Team> result = new ArrayList<>();
+    try{
+    reader = new CSVReader(new FileReader(fileName));}
+    catch (FileNotFoundException e) {
+        e.printStackTrace();
+        return null;
+    }
+    try {
+        strData = reader.readAll();
+    }catch (IOException e) {
+        e.printStackTrace();
+    } catch (CsvException e) {
```

We can't simply ignore this exception, if we have corrupted file that we can't read we should throw the error to indicate this issue to the user. 
this exception block can be merged with top one since the exception handling here is identical. Also see comment above about the logger.